example/app/config.yml

```diff
@@ -104,13 +107,34 @@ default:
           - cumulus-ecs-task
           - '--activityArn'
           - function: Ref
-            value: EcsTaskHelloWorldActivity
+            value: HelloWorldActivity
           - '--lambdaArn'
           - function: Ref
             value: HelloWorldLambdaFunction
+        service_scaling:
+          metric: CPUUtilization
+          scale_in:
+            evaluation_periods: 1
+          scale_out:
+            evaluation_periods: 1
+            step_adjustment_config:
+              AdjustmentType: PercentChangeInCapacity
+              StepAdjustments:
+              - MetricIntervalLowerBound: 0
```

Comment:
You expect these configurations are not helpful in that they scale too conservatively? In any event, this is more demonstrative than prescriptive since it's in `example/`